Tests/src/PublicAPITests/ECommerceSystemTests.cpp

```diff
@@ -157,6 +157,100 @@ CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetProductInformationTest)
 }
 #endif
 
+#if RUN_ECOMMERCE_TESTS || RUN_ECOMMERCE_GET_PRODUCT_INFORMATION_BY_VARIANT_TEST
+CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetProductInformationByVariantTest)
+{
+	/*Steps needed to be performed before running this test are:
+
+	1. Create a space (Add to Shopify Creds)
+	2. Connected your shopify.dev account to your space using the "Private Access Token" and store name
+		Endpoint : /api/v1/spaces/{spaceId}/vendors/shopify
+		{
+			"storeName": "string",
+			"isEcommerceActive": true,
+			"privateAccessToken": "string"
+		}
+	3. Check Shopify has synced with your namespace
+		Endpoint: /api/v1/vendors/shopify/validate
+		{
+			"storeName": "string",
+			"privateAccessToken": "string"
+		}
+	4. Either use the default "Gift Card" product or update these test variables with a new product. (Add variant Id to Shopify Creds)
+	Now you can use this test!*/
+	SetRandSeed();
+
+	auto& SystemsManager  = csp::systems::SystemsManager::Get();
+	auto* UserSystem	  = SystemsManager.GetUserSystem();
+	auto* ECommerceSystem = SystemsManager.GetECommerceSystem();
+
+	// This is an example from Shopify dev quickstart "Gift Card"
+	const csp::common::String ProductId				= "gid://shopify/Product/8566195847465";
```

Comment:
If my understanding is correct, the test will use 4 variant IDs of the same product, and expect that product to be returned. I think this is slightly different from what the API offers, as it doesn't specify the variant IDs have to be of the same product. 
 If my understanding is correct, I feel it's worth updating the test to take into account Variant IDs of differing products, and to expect those differing products back. That's because this would be something the client apps most likely will use it for.

Reply:
sounds good, thank you